cpp/include/cudf_test/cxxopts.hpp

```diff
@@ -552,13 +552,13 @@ class abstract_value : public Value {
 
   void parse(const std::string& text) const { parse_value(text, *m_store); }
 
-  bool is_container() const { return type_is_container<T>::value; }
+  bool is_container() const override { return type_is_container<T>::value; }
```

Comment:
This is vendored header. While these changes are innocuous, I don't think we should be changing it.

Reply:
Ill modify the script so this file is ignored.